response_operations_ui/templates/conversation-view/cv-reply-form.html

```diff
@@ -19,8 +19,20 @@
                             maxlength='10000',
                             cols=80) }}
         </div>
-        <input class="btn u-mt-s" id="btn-send-message" name="send" type="submit" value="Send" />
-        <a href="{{ url_for('messages_bp.close_conversation', thread_id=messages[0].thread_id, page=page) }}"
-            class="btn btn--secondary btn--border u-mt-s u-ml-xs" id="btn-close-conversation" >Close conversation</a>
+        {{
+            onsButton({
+                "text": "Send",
+                "classes": 'u-mt-s',
+                "id": "btn-send-message"
+            })
+        }}   
+        {{
+            onsButton({
```

Comment:
Does this remain as a link rather then a button?

Reply:
Yes. We pass a url into the button component.